forum/static/style.css

```diff
@@ -1,3 +1,49 @@
+@media(prefers-color-scheme: light){
+	body {
+		color: #888;
+		background: white;
+	}
+	.user_action > a {
+		color: white;
+	}
+	.smalltext{
+		color: black;
+	}
+	#footer {
+		background: white;
+	}
+	a {
+		color: blue
+	}
+}
+
+@media(prefers-color-scheme: dark){
+	body {
+		color: #888;
+		background: black;
```

Comment:
I think black is a bit too dark. Github uses `#131516` which I think is pretty nice.